src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
```

Comment:
Better to use an injector to get the Services
```suggestion
    private static final Injector injector = Injector.getInstance("mate.jdbc");
    private ManufacturerDao manufacturerDao = injector.getInstance(ManufacturerDao.class);
```